index.js

```diff
@@ -15,22 +15,15 @@ const bubleOpts = {
   }
 }
 
-const shouldSkip = file => contains(file, 'node_modules')
+const transformFile = function (filename, options = {}) {
+  const babelOptions = options.babel ? Object.assign({}, babelOpts, options.babel) : babelOpts
+  const bubleOptions = options.buble ? Object.assign({}, bubleOptions, options.buble) : bubleOpts
 
-const original = require.extensions['.js']
+  const { code: babelCode } = transformFileSync(filename, babelOptions)
```

Comment:
yeah i think the only thing i noticed was the errors were not as straightforward or had nice stack trace without a try/catch... because when i 1st set this package up i didnt use either